### PR TITLE
MM-12051 Use react-custom-scrollbars for System Console left-hand sidebar

### PR DESCRIPTION
**Pull request by niklabh**
#### Summary
- Replaces scrollbar with the react-custom-scrollbars for System Console
- Fixes issue where scrolling was unusably slow

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-12051

Fixes  https://github.com/mattermost/mattermost-server/issues/11956

### CODE REVIEW

**Comment by niklabh** [comment]
https://github.com/mattermost/mattermost-server/issues/11956

**Comment by niklabh** [comment]
removed

**Comment by niklabh** [comment]
@jespino anything else needs to be done here?

**Comment by jespino** [comment]
@niklabh I don't think so. You need to merge master in the branch to get everything up to date, but not needed to re-review it by my side.

File: components/admin_console/admin_sidebar/admin_sidebar.jsx [hunk 1]
```diff
@@ -284,46 +278,56 @@ export default class AdminSidebar extends React.Component {
...
-                                        <OverlayTrigger
-                                            delayShow={Constants.OVERLAY_TIME_DELAY}
-                                            placement='bottom'
-                                            overlay={filterClearTooltip}
+                <Scrollbars
+                    ref='scrollbar'
+                    autoHide={true}
+                    autoHideTimeout={500}
+                    autoHideDuration={500}
+                    renderThumbHorizontal={(props) => renderScrollView(props, 'scrollbar--horizontal')}
```
Comment (jespino) [inline components/admin_console/admin_sidebar/admin_sidebar.jsx]: We prefer to no regenerate functions in each render, do you mind to move this in independent functions like the `renderScrollView` but one per each class? I know Scrollbars is not a pure component so is going to re-render anyway, but we like to follow the policy to not generate bad examples in the code.

Comment (niklabh) [reply components/admin_console/admin_sidebar/admin_sidebar.jsx]: done

**Comment by jespino** [comment]
@niklabh lint failing because the unneded jquery import.

**Comment by niklabh** [comment]
removed unused $ import

**Comment by lindalumitchell** [comment]
@jespino does the linked server PR also need to be reviewed and go in at the same time this PR goes in? https://github.com/mattermost/mattermost-server/issues/11956

Also, this is fine with QA for it to go through without testing on the PR / new tests, so I'm removing the QA Review label.

**Comment by amyblais** [comment]
Removed from v5.16 because it didn't make it to feature complete. Please let me know if this is important for v5.16.

**Comment by hanzei** [comment]
@niklabh Would you please merge `master` into your branch?

**Comment by niklabh** [comment]
i don’t access to a computer till sunday. sorry

**Comment by hanzei** [comment]
No problem. Would have take care of this myself, but I can't push to your master branch.

**Comment by niklabh** [comment]
given you access

**Pull request merged**